lib/src/commands/radio.dart

```diff
@@ -84,6 +87,31 @@ ChatGroup radio = ChatGroup(
         await context.respond(MessageBuilder.embed(embed));
       }),
     ),
+    ChatCommand(
+      'recognise',
+      'Recognise the current song playing',
+      id('music-recognise', (
+        IChatContext context,
+      ) async {
+        try {
+          final recognitionService = SongRecognitionService.instance;
+          final guildId = context.guild?.id.toString() ?? '';
```

Comment:
This can never be null, as it's a command triggered from within a guild. A null-check operator would be okay here!